Makefile

```diff
@@ -48,7 +48,11 @@ diff-commit: $(draft).txt $(draft)-$(COMMIT).txt
 $(next).md: $(draft).md
 	sed -e"s/$(basename $<)-latest/$(basename $@)/" $< > $@
 
-%.xml: %.md
+%.md2: %.md
+	- rm $@
```

Comment:
You shouldn't have to use rm here.  Try adding
.INTERMEDIATE: %.md2

And add *.md2 to your clean target.